client/css/reviewRating.scss

```diff
@@ -32,34 +34,36 @@
   width: 80%;
 }
 .rating_bar{
-  border: 1px solid black;
-  background-color: rgb(199, 197, 197);
+  background-color: white;
```

Comment:
More often than not, for colors like black and white, you're going to see RGB hex values like #000 for black and #fff for white.

Reply:
Thanks, good to know, I would never use that if you don't tell me.